pxl_scripts/px/services_to_ips/services_to_ips.pxl

```diff
@@ -0,0 +1,36 @@
+# Finds services that have traffic to and from a list of IP addresses.
```

Comment:
Also want to add a manifest.yaml. You can copy it from another script and edit it to describe this one.